### DISCUSSION TITLE
Video thumbnails size

### DISCUSSION THREAD

**Discussion by alghazanth** [sxyazi/yazi#1315] [1. Q&A] [answered]
Hello! Is it possible somehow to set specific size of video thumbnails generated by ffmpegthumbnailer?

**Comment by sxyazi** [comment, marked as answer]
Its controled by both `max_width` and `max_height`, https://yazi-rs.github.io/docs/configuration/yazi#preview.max_width

**Reply by alghazanth** [reply]
Thank you a lot! I've read this section of the documentation, but was confused by description of this setting "Maximum preview width for **images**".
